packages/tjdashboard/administrator/views/dashboard/view.html.php

```diff
@@ -57,6 +57,12 @@ class TjdashboardViewDashboard extends JViewLegacy
 	 */
 	public function display($tpl = null)
 	{
+		require_once JPATH_COMPONENT . '/../com_tjlms/helpers/tjlms.php';
```

Comment:
@monikaatvowel Remove this from tjdashboard

Reply:
now removed